File: procedural-programming/homework-001/python/exercise_038.py
# Student: Pedro Henrique Resende Ribeiro
# Description: Print an employee's new salary
# Date: 2024-01-09 - Time: 01:50

salary = float(input("Enter the employee's salary: "))

increase = 0.25
newSalary = salary * (1 + increase)

print(f'The new salary is {newSalary:,.2f}')

File: procedural-programming/homework-001/c/exercise-038.c
/* Student: Pedro Henrique Resende Ribeiro
 * Description: Print an employee's new salary
 * Date: 2024-01-09 - Time: 01:50
 */

#include <stdio.h>

int main() {

    float salary, increase, newSalary;

    printf("Enter the employee's salary: ");
    scanf("%f", &salary);

    increase = 0.25;
    newSalary = salary * (1 + increase);

    printf("The new salary is %.2f\n", newSalary);

    return 0;

}
